### DISCUSSION TITLE
Problem with transformers package

### DISCUSSION THREAD

**Discussion by alfeilex** [PromtEngineer/localGPT#115] [Q&A]
Hello, I have this problem after starting the query:

```
C:\Users\****\AppData\Local\Programs\Python\Python311\Lib\site-packages\transformers\generation\utils.py:1255: UserWarning: You have modified the pretrained model configuration to control generation. This is a deprecated strategy to control generation and will be removed soon, in a future version. Please use a generation configuration file (see https://huggingface.co/docs/transformers/main_classes/text_generation)
  warnings.warn(
```

What does that mean, do you have any idea what cause this problem?

**Comment by PromtEngineer** [comment]
This is a warning, so the code will still work without any issues. This is because of how the model is pre-trained. We can look into updating it at some point but probably we will not have this issue with newer models that I plan to add. Check [this](https://github.com/huggingface/transformers/blob/main/src/transformers/generation/utils.py).
